### PR TITLE
Fixed doc missed

### PR DESCRIPTION
**Pull request by vincenzopalazzo**
All commands without documentation are

~There are a couple of other commands such as `stop`, `ping`, `listconfigs`, `getlog`, `getinfo`, but I don't think these need a documentation file~

The complete list here 
```
   "commands": [
      "getinfo",
      "getlog",
      "help",
      "listconfigs",
      "listnodes",
      "ping",
      "sendpsbt",
      "signpsbt",
      "stop"
   ]
```

## Update 

The complete list of the commands is.

 - [X] listtransactions
 - [X] listnodes
 - [X] signpsbt
 - [X] sendpsbt
 - [X] getinfo
 - [X] getlog
 - [X] help
 - [X] listconfigs
 - [X] ping
 - [X] stop

### CODE REVIEW

File: doc/lightning-listtransactions.7.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      }
+    ]
+}
+```
+
+
+AUTHOR
+------
+
+Vincenzo Palazzo <<vincenzo.palazzo@protonmail.com>> wrote the initial version of this man page, but many others did the hard work of actually implementing of this rpc command.
```
Comment (ZmnSCPxj) [inline doc/lightning-listtransactions.7.md:90]: s/actually implementing of/actually implementing/

File: doc/lightning-listtransactions.7.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  "id": 82,
+  "method": "listtransactions",
+  "params": {}
+}
+```
+
+RETURN VALUE
+------------
+
+On success, the command will return a list of transactions, each object rappresent the transaction with all details.
```
Comment (ZmnSCPxj) [inline doc/lightning-listtransactions.7.md:27]: s/rappresent the transaction with all details./represents a transaction, with the following details:/

File: doc/lightning-listtransactions.7.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+- *hash*: A string that rappresent  the hash of transaction, with the caller can use to find it on blockchain.
```
Comment (ZmnSCPxj) [inline doc/lightning-listtransactions.7.md:29]: s/rappresent /represents/

s/with the caller/which the caller/

s/on blockchain/on the blockchain/

File: doc/lightning-listtransactions.7.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+- *rawtx*: A string that rappresent the hexdecimal of transaction.
```
Comment (ZmnSCPxj) [inline doc/lightning-listtransactions.7.md:30]: s/rappresent/represents/

s/hexdecimal of transaction/hexadecimal dump of the transaction/

File: doc/lightning-listtransactions.7.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+- *blockheight*: An integer that rappresent the block height that contains the transaction on blockchain.
```
Comment (ZmnSCPxj) [inline doc/lightning-listtransactions.7.md:31]: %s/rapprasent/represents/g

File: doc/lightning-listtransactions.7.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+- *txindex*: An integer that rappresent the transaction index inside the block.
+- *locktime*: An integer that rappresent the locktime.
+- *version*: An integer that rappresent the version.
+- *inputs*: A list of transactions, each transaction is rappresented with an object with the following proprieties:
```
Comment (ZmnSCPxj) [inline doc/lightning-listtransactions.7.md:35]: s/A list of transactions, each transaction/A list of spent transaction outputs, each spent transaction output/

s/proprieties/properties/

File: doc/lightning-listtransactions.7.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+  - *txid*: A string that rappresent the hash of transaction.
+  - *index*: An integer that rappresent the index of transaction.
```
Comment (ZmnSCPxj) [inline doc/lightning-listtransactions.7.md:37]: This is the output index of the transaction output being spent.

File: doc/lightning-listtransactions.7.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+  - *sequence*: A integera that rappresent the sequence.
```
Comment (ZmnSCPxj) [inline doc/lightning-listtransactions.7.md:38]: s/A integera/An integer/

s/the sequence/the sequence number/

File: doc/lightning-listtransactions.7.md [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+- *outputs*: A list of transactions, each transaction is rappresented with an object with the following proprieties:
+  - *index*: An integer that rappresent the index of transaction.
```
Comment (ZmnSCPxj) [inline doc/lightning-listtransactions.7.md:40]: This is the output index of the transaction output.

File: doc/lightning-listtransactions.7.md [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+- *rawtx*: A string that rappresent the hexdecimal of transaction.
+- *blockheight*: An integer that rappresent the block height that contains the transaction on blockchain.
+- *txindex*: An integer that rappresent the transaction index inside the block.
+- *locktime*: An integer that rappresent the locktime.
+- *version*: An integer that rappresent the version.
+- *inputs*: A list of transactions, each transaction is rappresented with an object with the following proprieties:
+  - *txid*: A string that rappresent the hash of transaction.
+  - *index*: An integer that rappresent the index of transaction.
+  - *sequence*: A integera that rappresent the sequence.
+- *outputs*: A list of transactions, each transaction is rappresented with an object with the following proprieties:
```
Comment (ZmnSCPxj) [inline doc/lightning-listtransactions.7.md:39]: This is a transaction output as well.

File: doc/lightning-listtransactions.7.md [hunk 11]
```diff
@@ -0,0 +1,100 @@
...
+  - *index*: An integer that rappresent the index of transaction.
+  - *satoshis*: A string that rappresent the amount in millisatoshi that contains the transaction.
+  - *scriptPubKey*: A string that contains the lock script.
```
Comment (ZmnSCPxj) [inline doc/lightning-listtransactions.7.md:42]: in hexadecimal dump form.

File: doc/lightning-listtransactions.7.md [hunk 12]
```diff
@@ -0,0 +1,100 @@
...
+  
+On failure, an error is returned and any result was returned.
```
Comment (ZmnSCPxj) [inline doc/lightning-listtransactions.7.md:44]: Simplify this:

```
On failure, one of the following error codes may be returned:
* -32602. Error in given parameters.
```

File: doc/lightning-listtransactions.7.md [hunk 13]
```diff
@@ -0,0 +1,100 @@
...
+
+AUTHOR
+------
+
+Vincenzo Palazzo <<vincenzo.palazzo@protonmail.com>> wrote the initial version of this man page, but many others did the hard work of actually implementing of this rpc command.
+
+SEE ALSO
+--------
+
+FIXME: add somethings.
```
Comment (ZmnSCPxj) [inline doc/lightning-listtransactions.7.md:95]: Suggest: lightning-newaddr(7), lightning-listfunds(7)

Those docs might want to add lightning-listtransactions(7) in their own "SEE ALSO" sections.

**Comment by vincenzopalazzo** [comment]
Rebase https://github.com/ElementsProject/lightning/pull/3938/commits/169967f3e4fd3b18984dab6ae32ae6e675d9dae5

In addition, I have some problems with the command `mrkd`.

If I run the command `mrkd doc/lightning-signpsbt.7.md doc/lightning-signpsbt.7 ` I have the following error

```
Traceback (most recent call last):
  File "/usr/local/bin/mrkd", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.6/dist-packages/mrkd.py", line 254, in main
    plac.call(entry_point)
  File "/usr/local/lib/python3.6/dist-packages/plac_core.py", line 417, in call
    cmd, result = parser.consume(arglist)
  File "/usr/local/lib/python3.6/dist-packages/plac_core.py", line 276, in consume
    return cmd, self.func(*(args + varargs + extraopts), **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/mrkd.py", line 225, in entry_point
    result = mistune.markdown(fp.read(), inline=inline, renderer=renderer)
  File "/usr/local/lib/python3.6/dist-packages/mistune.py", line 1184, in markdown
    return Markdown(escape=escape, **kwargs)(text)
  File "/usr/local/lib/python3.6/dist-packages/mistune.py", line 994, in __call__
    return self.parse(text)
  File "/usr/local/lib/python3.6/dist-packages/mistune.py", line 1004, in parse
    out = self.output(preprocessing(text))
  File "/usr/local/lib/python3.6/dist-packages/mistune.py", line 1053, in output
    out += self.tok()
  File "/usr/local/lib/python3.6/dist-packages/mistune.py", line 1063, in tok
    return getattr(self, 'output_%s' % t)()
  File "/usr/local/lib/python3.6/dist-packages/mistune.py", line 1125, in output_list
    body += self.tok()
  File "/usr/local/lib/python3.6/dist-packages/mistune.py", line 1063, in tok
    return getattr(self, 'output_%s' % t)()
  File "/usr/local/lib/python3.6/dist-packages/mistune.py", line 1134, in output_list_item
    body += self.tok()
  File "/usr/local/lib/python3.6/dist-packages/mistune.py", line 1063, in tok
    return getattr(self, 'output_%s' % t)()
  File "/usr/local/lib/python3.6/dist-packages/mistune.py", line 1126, in output_list
    return self.renderer.list(body, ordered)
  File "/usr/local/lib/python3.6/dist-packages/mrkd.py", line 84, in list
    assert body[0] == '\0'
IndexError: string index out of range
```

I'm losing something?

**Comment by rustyrussell** [comment]
Change `- -32602.` to `- -32602:`.

**Comment by vincenzopalazzo** [comment]
Ops! Thanks to have fix that @rustyrussell

**Comment by rustyrussell** [comment]
I did another sweep on top of this one, and hope that I found all remaining issues!

**Comment by rustyrussell** [comment]
Ack https://github.com/ElementsProject/lightning/pull/3938/commits/8f1deedd324ff8818a4b4f15c31a69c814af86c0

**Pull request merged**